release-notes.md

```diff
@@ -137,6 +142,7 @@ Add the following lines as new sections:
 	 - Password: kermit
  - Generate java pojo and bpmn clients **(replace tokens)**: `php @APP_DIR@/generator/generate.php pojo,bpmn`
  - Edit deployment configuration file **(replace tokens)**: `cp @WEB_DIR@/content/clientlibs/bpmn/deploy/src/activiti.cfg.template.xml @WEB_DIR@/content/clientlibs/bpmn/deploy/src/activiti.cfg.xml`
+ - Download [XercesImpl JAR] (http://central.maven.org/maven2/xerces/xercesImpl/2.8.1/xercesImpl-2.8.1.jar) and extract contents to @WEB_DIR@/content/clientlibs/bpmn/deploy/src/
```

Comment:
We can download the jar to @WEB_DIR@/content/clientlibs/bpmn/deploy/lib without the need to extract it. Or even better, you can commit it under  generator/sources/bpmn/deploy/lib just like mysql-connector-java-5.0.8-bin.jar and then no manual actions will be needed.